pyoverkiz/enums/measured_value_type.py

```diff
@@ -0,0 +1,43 @@
+from enum import Enum, unique
+
+
+@unique
+class MeasuredValueType(str, Enum):
+    """See https://github.com/Somfy-Developer/Somfy-TaHoma-Developer-Mode/issues/68#issuecomment-1281474879"""
+
+    ABSOLUTE_VALUE = "core:AbsoluteValue"
+    ANGLE_IN_DEGREES = "core:AngleInDegrees"
+    ANGULAR_SPEED_IN_DEGREES_PER_SECOND = "core:AngularSpeedInDegreesPerSecond"
+    ELECTRICAL_ENERGY_IN_KWH = "core:ElectricalEnergyInKWh"
+    ELECTRICAL_ENERGY_IN_WH = "core:ElectricalEnergyInWh"
+    ELECTRICAL_POWER_IN_KW = "core:ElectricalPowerInKW"
+    ELECTRICAL_POWER_IN_W = "core:ElectricalPowerInW"
+    ELECTRIC_CURRENT_IN_AMPERE = "core:ElectricCurrentInAmpere"
+    ELECTRIC_CURRENT_IN_MILLI_AMPERE = "core:ElectricCurrentInMilliAmpere"
+    ENERGY_IN_CAL = "core:EnergyInCal"
+    ENERGY_INK_CAL = "core:EnergyInkCal"
```

Comment:
```suggestion
    ENERGY_IN_KCAL = "core:EnergyInkCal"
```

Would this be kilocalorie?

Reply:
I would say yes.